app/reducers/cart.jsx

```diff
@@ -37,3 +38,29 @@ export const RemoveFromCart = magnetId => ({
 export const ChangeItemQuantity = magnetWithQuant => ({
   type: CHANGE_ITEM_QUANTITY, magnetWithQuant
 })
+
+export const addToOrder = magnetId => dispatch =>
+    axios.get(`/api/magnets/${magnetId}`)
```

Comment:
Seems like we can just get the magnet from our store state by matching the ids, right?